source/assets/locales/en.json

```diff
@@ -168,7 +168,10 @@
     "congratulations": "CONGRATULATIONS!"
   },
   "import": {
-    "importingYourAccount": "Importing your wallet seed automatically import a wallet associated with this seed phrase.",
+    "importModalWordMissing": "Word missing!",
+    "importModalTheSeedPgrase": "The seed phrase must be 12 or 24 words and you inserted 11.",
```

Comment:
IMO, _**"ImportmodaltheedPgraSe"**_ establishes an error message saying that you have inserted 11 words.

I have not seen a function that interpreted that if the amount of words is less than 12, and if it is greater than twelve and less than 24, to determine if the user is introducing a Seed of 12 or 24 or so show this message.

However, an amount of Seeds definite as _"you have entered 11"_, when the user by mistakes enter 10 words, I do not see a method of counting quantity. The text would always show _"you have introduced 11"_, therefore the amount of words introduced should be controlled by this variable that counts the amount of Seeds introduced by user.

Another option is to simply omit this complication of code editing, and simply show a generic message telling the user that some Seeds is missing, or that the amount of Seeds is incorrect and reviewed.

Reply:
Sure, but this PR has not yet been finalized. I'm already adjusting these points, I had just opened the PR for my control. But thanks for the points anyway!!!